server/zanata-frontend/src/app/editor/index.html

```diff
@@ -19,7 +19,7 @@
   <link rel="icon" type="image/x-icon" href="http://zanata.org/images/logo/logo.ico">
   <link rel="stylesheet" type="text/css" href="/editor.css">
   <link rel="stylesheet" type="text/css" async
-    href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic'>
+    href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic'>
```

Comment:
How did you decide whether to change `http` -> `https` or to remove the protocol entirely? Most of the other changes omit the protocol.

Actually, reading a bit more about it (eg https://jeremywagner.me/blog/stop-using-the-protocol-relative-url/) I think we should unconditionally use `https`, both for security and speed reasons (eg HTTP2). 

Sorry to change things on you.

Reply:
No problem. Thought we should be using https where possible.

Reply:
What I mean is, could you please change all five of the `//www.gravatar.com` links to `https//www.gravatar.com` too? There's four of them in this PR, but there's one in src/index.html too.

Reply:
No problem. Sorry for the misunderstanding